libs/state/actions/src/lib/inject.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  ChangeDetectorRef,
+  ErrorHandler,
+  ViewRef,
+  inject,
+} from '@angular/core';
+import { RxActionFactory } from './actions.factory';
+
+export function injectRxActionFactory<
```

Comment:
should it be named `rxActionFactory`, or even simpler `rxActions` ? Why do we even have to call the create function as a user?

Reply:
We would need to provide a `withTransforms` function in order to pass input transforms to `actions.create(transforms)` but yes it would simplify the boilerplate and align with composition functions in `rxState`.